app/src/styles.sass

```diff
@@ -34,10 +34,7 @@ input
     @include userbox
     background-color: $userbuttonbgcolor
     font-size: $fontsize
-    margin-left: $medium-space
```

Comment:
No. This margin is mandatory in most cases. You must set it to zero only in the login and signup forms.

Reply:
ah I see